DSharpPlus/Entities/DiscordMember.cs

```diff
@@ -268,7 +274,61 @@ public DiscordColor Color
         /// <returns></returns>
         public Task ReplaceRolesAsync(IEnumerable<DiscordRole> roles, string reason = null) =>
             this.Discord.ApiClient.ModifyGuildMemberAsync(this.Guild.Id, this.Id, null, roles.Select(xr => xr.Id), null, null, null, reason);
+
```

Comment:
eh... ehhhhhh... not sure about these, ask Naam.

Reply:
Seems good to me, honestly. I've originally added the guild object to not have to look up the guild to do some guild-specific operations.

Adding links only makes this easier